### DISCUSSION TITLE
How do you suggest handling manifest.json?

### DISCUSSION THREAD

**Discussion by james0r** [barrel/shopify-vite#144] [Q&A] [answered]
It looks like the manifest isn't being used anywhere. I've just been rendering it to it's default `assets/.vite/manifest.json` and then ignoring it within my .shopifyignore.

How are you handling this?

**Comment by montalvomiguelo** [comment, marked as answer]
I'm doing the same. The manifest file is used to build the snippet that loads scripts and styles during production mode. You could also add it to your .gitignore to avoid tracking it in your repository.

**Reply by james0r** [reply]
Gotcha. I'm gitignore'ing my assets file recursively so it'll include the manifest. Any reason I shouldn't be?
